src/main/resources/templates/footer.html

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>Footer</title>
+
+</head>
+<body>
+<footer class="text-center text-white" style="background-color: rgba(0, 0, 0, 0.2);" th:fragment="footer">
```

Comment:
Adding a `style` attribute is discouraged. This should be used only as a last resort.
We should use cascading style sheets whenever possible.

Reply:
Created style.css file in which was added the background color.